### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by vasilty**
The reason why this fails might deserve a coffee break

### CODE REVIEW

**Comment by i-trofimtschuk** [comment]
will wait for coffee break until i can approve, since without explanations it's hard to follow why this change makes the test pass

**Review by guettli — APPROVED** [review]

File: tests/test_workerprocess.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import multiprocessing
+import multiprocessing.connection
```
Comment (i-trofimtschuk) [inline tests/test_workerprocess.py:2]: wow, ow did it work before without?

**Review by i-trofimtschuk — APPROVED** [review]
